### ISSUE TITLE
Thank you !!

### ISSUE THREAD

**Issue by biovolt** [clockbyte/admobadapter#6]
Could you please ad the option to display a ad in the 10th 20th and the the last index if there are more then 28 ads ? 

e.g. if there are 32 ads
Index if ads (Including ads) 10,20,33? 

Also I get a flickering each time when the AdmobListener onAdCountChanged is called.
Is there any way to avoid the flickering when an ad is fetched?

**Comment by clockbyte** [comment]
Hi!
On flickering: Do you use Express ads?

On last index: Am I right that you want to have an option to display an ad block at the last position? i.e. you have 28 items in the list and you've set to show 3 ad blocks in each 10 items, then list should present the ad blocks at 10th, 20th and finally at 28th index?

**Comment by biovolt** [comment]
Hi
With each onContentAdLoaded callback the list is updated and redrawn. This caused the image to be updated and re fetched the image :-(

So I had to check if it is a the data set on convertView matched the data in the adapter for the view.
so that I only update it if it changed.

Don't worry about the positions. We have different ad placements (AdUnitIds) for different positions so that the ad for position 10 can be targeted different than the one at the end one at the end of the list.

Yes for the last ad at index 28.if the list is 45 would yield   [10, 20, 45]
The problem is, because the ads are targeted for the positions and  e.g. the middle ad fail the last ad should still be loaded at the last position.

We have started reimplementing it using a interface to avoid the whole conversion between ad id and item id.

Thank you for your contribution it really helped me !
